src/main/java/cardibuddy/model/deck/Deck.java

```diff
@@ -70,6 +70,15 @@ public boolean equals(Object other) {
                 && otherDeck.getFlashcards().equals(getFlashcards());
     }
 
+    public boolean isSameDeck(Deck otherDeck) {
```

Comment:
what's the need for this one! compared to equals(). Are there instances where the deck contents are different but the title same?

Reply:
removed equals!